frontend/src/services/graphQLDevice.ts

```diff
@@ -64,6 +64,59 @@ const DEVICE_SELECT = `
         email
       }
     }
+  }
```

Comment:
Let's not just get these logs when we get the device list, instead break these queries out into their own graphQLLogs.ts file and let's get them on demand when we go to the page. That will be much better for device list load times.